nums/numpy/api.py

```diff
@@ -27,15 +27,29 @@
 # pylint: disable = redefined-builtin, too-many-lines
 
 
-def _not_implemented(fun):
+def _not_implemented(func):
     # From project JAX: https://github.com/google/jax/blob/master/jax/numpy/lax_numpy.py
     def wrapped(*args, **kwargs):
         # pylint: disable=unused-argument
         msg = "NumPy function {} not yet implemented."
-        raise NotImplementedError(msg.format(fun))
+        raise NotImplementedError(msg.format(func))
     return wrapped
 
 
+def _default_to_numpy(func):
```

Comment:
Can you add a TODO here to later convert this function to invoke the numpy op on a worker vs. the driver?

Reply:
Sure.